README-MALAY.md

```diff
@@ -0,0 +1,142 @@
+- [Getting into Open Source for the First Time](https://www.nearform.com/blog/getting-into-open-source-for-the-first-time/) karya [@mcdonnelldean](https://github.com/mcdonnelldean)
+- [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/) karya [@GitHub](https://github.com/github)
+- [How to Find a Bug in Your Code](https://8thlight.com/blog/doug-bradbury/2016/06/29/how-to-find-bug-in-your-code.html) karya [@dougbradbury](https://twitter.com/dougbradbury)
+- [Mastering Markdown](https://guides.github.com/features/mastering-markdown/) karya [@GitHub](https://github.com/github)
+- [First mission: Contributors page](https://medium.com/@forCrowd/first-mission-contributors-page-df24e6e70705#.2v2g0no29) karya [@forCrowd](https://github.com/forCrowd)
+- [How to make your first Open Source contribution in just 5 minutes](https://www.freecodecamp.org/news/how-to-make-your-first-open-source-contribution-in-just-5-minutes-aaad1fc59c9a/) karya [@roshanjossey](https://github.com/Roshanjossey/)
+- [Hacktoberfest 2019: How you can get your free shirt — even if you’re new to coding](https://www.freecodecamp.org/news/hacktoberfest-2018-how-you-can-get-your-free-shirt-even-if-youre-new-to-coding-96080dd0b01b/) karya [@quincylarson](https://www.freecodecamp.org/news/author/quincylarson/)
+- [A Bitter Guide To Open Source](https://medium.com/codezillas/a-bitter-guide-to-open-source-a8e3b6a3c1c4) karya [@ken_wheeler](https://medium.com/@ken_wheeler)
+- [A junior developer’s step-by-step guide to contributing to Open Source for the first time](https://hackernoon.com/contributing-to-open-source-the-sharks-are-photoshopped-47e22db1ab86) karya [@LetaKeane](https://hackernoon.com/u/letakeane)
+- [Learn Git and GitHub Step By Step (on Windows)](https://medium.com/illumination/path-to-learning-git-and-github-be93518e06dc) karya [@ows-ali](https://medium.com/@ows_ali)
+- [Why Open Source and How?](https://careerkarma.com/blog/open-source-projects-for-beginners/) karya [@james-gallagher](https://careerkarma.com/blog/author/jamesgallagher/)
+- [How to get started with Open Source - By Sayan Chowdhury](https://www.hackerearth.com/getstarted-opensource/)
+- [What open-source should I contribute to](https://kentcdodds.com/blog/what-open-source-project-should-i-contribute-to/) karya Kent C. Dodds
+
+## Menggunakan Pengawalan Versi
+- [Think Like (a) Git](http://think-like-a-git.net/) - Pengenalan Git untuk "pemula maju", tetapi masih berusaha, untuk memberi anda strategi mudah untuk bereksperimen dengan selamat dengan git.
+- [Try Git](https://try.github.io/) - Belajar Git dalam 15 minit dalam laman anda.
+- [Everyday Git](https://git-scm.com/docs/giteveryday) - Satu set perintah minimum yang berguna untuk Everyday Git.
+- [Oh shit, git!](https://ohshitgit.com/) - cara mengatasi kesalahan `git` yang biasa dijelaskan dalam bahasa Inggeris biasa; lihat juga [Dangit, git!](https://dangitgit.com/) untuk halaman tanpa sumpah.
```

Comment:
- remove `biasa` before dijelaskan. So, it will becomes `yang dijelaskan`
- change `sumpah` to `carutan`